src/Hill.jl

```diff
@@ -134,3 +134,38 @@ function plotUnitSensitivity(paramRange, result, realParam, i)
     plot!([realParam], seriestype = "vline", margin = 0.3cm, legend = :false)
     ylims!((1E2, 1E4))
 end
+
+
+""" Calculate the # of cells in G1 for a set of parameters and T """
+function numcells(params, g0, T)
+    t = LinRange(0.0, 95.5, 3)
+    temp = copy(params)
```

Comment:
Is there a reason you need to copy params?

Reply:
Oh this has remained from before that I wanted to sweep the parameters. I will remove it.